pkg/mcs/resourcemanager/server/token_buckets.go

```diff
@@ -108,17 +114,21 @@ func (gts *GroupTokenBucketState) resetLoan() {
 	}
 
 	evenTokens := gts.Tokens * evenRatio
+	now := time.Now()
+	gts.lastCheckExpireSlot = now
```

Comment:
Do we need to check expiration here first?

Reply:
deleted, no need to update here